open-metadata-implementation/adapters/open-connectors/repository-services-connectors/search-indexing-connectors/pom.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--SPDX-License-Identifier: Apache-2.0-->
+<!--Copyright Contributors to the ODPi Egeria project.-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <parent>
+        <artifactId>open-metadata-implementation</artifactId>
+        <groupId>org.odpi.egeria</groupId>
+        <version>1.5-SNAPSHOT</version>
+        <relativePath>../../../../pom.xml</relativePath>
```

Comment:
The parent should normally be the default - ie one level up (..) -

Reply:
The parent is still wrong in my view - why are we using open-metadata-implementation?

We should follow the pattern adaopted under open-metadata-archive-connectors where we actually have a pom (of type packaging=pom) for each level, so as you traverse each directory down we specify as a module in it's parent, and also refer to the parent from the child module. 

So we need this for search-indexing-connectors for example

Perhaps there's a better way - but if so needs explanation, proposal, and aim to be consistent?